test/lint/lint-github-actions.py

```
@@ -0,0 +1,117 @@
+    # packages:write at workflow level is required because reusable workflows
+    # (build-container.yml) inherit caller permissions and need it to push to ghcr.io
+    'excessive-permissions': [
+        'build.yml:9',
+    ],
+}
+
+
+def check_zizmor_install():
+    try:
+        subprocess.run(
+            ['zizmor', '--version'],
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)